x-pack/solutions/security/plugins/cloud_security_posture/public/components/no_findings_states/no_findings_states.test.tsx

```diff
@@ -171,7 +171,7 @@ describe('NoFindingsStates', () => {
         screen.getByText(/required elasticsearch index privilege for the following indices:/i)
       ).toBeInTheDocument();
       expect(
-        screen.getByText('logs-cloud_security_posture.findings_latest-default')
+        screen.getByText('logs-cloud_security_posture.misconfiguration_latest')
```

Comment:
Shouldn't it be `security_solution-cloud_security_posture.misconfiguration_latest` ? Did this test pass? if yes, does it check what we think it should check?